assets/js/atomic/blocks/product/add-to-cart/product-types/variable/variation-attributes/attribute-select-control.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { decodeEntities } from '@wordpress/html-entities';
+import { SelectControl } from '@wordpress/components';
+
+// Default option for select boxes.
+const selectAnOption = {
+	value: '',
+	label: __( 'Select an option', 'woo-gutenberg-products-block' ),
+};
+
+/**
+ * VariationAttributeSelect component.
+ *
+ * @param {*} props Component props.
+ */
+const AttributeSelectControl = ( {
+	attributeName,
+	options = [],
+	selected = '',
+	onChange = () => {},
+} ) => {
+	return (
+		<SelectControl
+			className="wc-block-components-product-add-to-cart-attribute-picker-select"
```

Comment:
And this one: `wc-block-components-product-add-to-cart__attribute-picker-select`.